ccmlib/cluster.py

```diff
@@ -106,36 +106,78 @@ def set_install_dir(self, install_dir=None, version=None, verbose=False):
 
     def actively_watch_logs_for_error(self, on_error_call, interval=1):
```

Comment:
This only scans system.log yes? Can you make that explicit inside the docstring?